Trend Micro/trend-micro-vision-one-oat/_meta/fields.yml

```diff
@@ -3,6 +3,46 @@ action.properties.ScriptBlockText:
   name: action.properties.ScriptBlockText
   type: keyword
 
+email.attachments:
+  description: A list of objects describing the attachment files sent along with an
+    email message
+  name: email.attachments
+  type: array
+
+email.delivery_timestamp:
+  description: The date and time when the email message was received by the service
+    or client
+  name: email.delivery_timestamp
+  type: date
+
+email.from.address:
+  description: 'The email address of the sender, typically from the RFC 5322 From:
+    header field'
+  name: email.from.address
+  type: keyword
```

Comment:
I agree with @TOUFIKIzakarya. You don't need to declare these fields. They must be already declared in the ingestion engine (Otherwise, there is an issue).